README.md

```diff
@@ -389,30 +389,23 @@ await sql`
 Use cursors if you need to throttle the amount of rows being returned from a query. New results won't be requested until the promise / async callback function has resolved.
 
 ```js
-
-await sql`
-  select * from generate_series(1,4) as x
-`.cursor(async ([row]) => {
+for await (const row of sql`select * from generate_series(1,4) as x`.cursor()) {
```

Comment:
It's also always a result array being returned, even if a single row, so it needs to be `const [row] of ...`

Reply:
Shoot I missed that lol

Reply:
Hehe.. that's on me for just making these things with no traces ;)